@stellar/anchor-tests/README.md

```diff
@@ -0,0 +1,69 @@
+# Stellar Anchor Tests
+
+`stellar-anchor-tests` is a library and command line tool for testing Stellar
+anchors.
+
+## Install
+
+This project is in active development and is not yet published to the npm
+registry. To install, run the following commands:
+
+```
+git clone git@github.com:stellar/stellar-anchor-tests.git
+yarn build
+yarn link
```

Comment:
this is mostly moving the README into this subfolder, but please note this change:

we need to use `yarn link` instead of `npm link` when using yarn workspaces. `npm link` won't properly link dependencies

Reply:
I originally tried using `yarn link` but had issues locally. It seemed to have created an invalid symlink, so when I tried running the `stellar-anchor-tests` command it would give me a command not found error. If it works fine for you then I'm sure I can figure it out 👍